pkg/plugins/directory/index.json

```
@@ -0,0 +1,8 @@
+[
+  {
+    "name": "azure",
+    "author": "Porter Authors",
+    "description": "A plugin for utilizing Azure cloud resources",
```

Let's change the description to make this a bit more clear on how the plugin helps. If you copied that from somewhere, can you let me know so that I can fix it there too?

```suggestion
    "description": "Integrate Porter with Azure. Store Porter's data in Azure Cloud and secure your bundle's secrets in Azure Key Vault.",
```